auth/permisos.ts

```diff
@@ -88,7 +89,8 @@ export default [
                     { key: 'solicitud', title: 'Registrar solicitud', type: 'boolean' },
                     { key: 'autocitado', title: 'Registrar Autocitación', type: 'boolean' },
                     { key: 'darTurnos', title: 'Dar turnos desde punto inicio', type: 'boolean' },
-                    { key: 'puedeEditarCarpeta', title: 'Editar número de carpeta', type: 'boolean' }
+                    { key: 'puedeEditarCarpeta', title: 'Editar número de carpeta', type: 'boolean' },
+                    { key: 'READ', title: 'Solo Lectura', type: 'boolean' }
```

Comment:
Mover arriba de todo así en el gestor de usuarios se ve primero.

```suggestion
                    { key: 'read', title: 'Ver turnos', type: 'boolean' }
```

Reply:
@liquid36 corregido